guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -41,26 +41,16 @@ endif::[]
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *Register Host*.
-. Optional: Select a different *Organization*.
-. Optional: Select a different *Location*.
-. Optional: From the *Host Group* list, select the host group to associate the hosts with.
-Fields that inherit value from *Host group*: *Operating system*, *Activation Keys* and *Lifecycle environment*.
-+
-If your host group has any Ansible roles assigned, the Ansible roles will run on your host upon the registration.
-. Optional: From the *Operating system* list, select the operating system of hosts that you want to register.
-ifndef::satellite,orcharhino[]
-Specifying an operating system is required when you register machines without `subscription-manager`, such as Debian or Ubuntu.
-endif::[]
-. Optional: From the *{SmartProxy}* list, select the {SmartProxy} to register hosts through.
-+
-[NOTE]
-====
-A {SmartProxy} behind a load balancer takes precedence over a {SmartProxy} selected in the {ProjectWebUI} as the host's content source.
-====
-ifdef::katello,satellite,orcharhino[]
-. In the *Activation Keys* field, enter one or more activation keys to assign to hosts.
-endif::[]
-. Optional: Select the *Insecure* option, if you want to make the first call insecure.
+. Enter the details for how you want the registered hosts to be configured.
+. Click *Generate*.
+. Run the `curl` command as `root` on the host that you want to register.
+After registration completes, any Ansible roles assigned to a host group you specified when configuring the registration template will run on the host.
+
+The registration details that you can specify include the following:
+
+* Optional: On the *General* tab, in the *{SmartProxy}* field, select the {SmartProxy} to register hosts through.
+A {SmartProxy} behind a load balancer takes precedence over a {SmartProxy} selected in the {ProjectWebUI} as the content source of the host.
+* Optional: On the *General* tab, select the *Insecure* option to make the first call insecure.
 During this first call, hosts download the CA file from {Project}.
 Hosts will use this CA file to connect to {Project} with all future calls making them secure.
 +
```

Comment:
I know this is outside the scope of your changes, but it looks like the CA file manual copy/install is a separate procedure. You might want to break it out so that it has its own procedure module if it does not already exist.

If the procedure is used frequently, it could be turned into a snippet. Your call.

Reply:
I agree, the description doesn't look right within the procedure. But you are right, it's out of scope of this PR. Let's file an issue.

Reply:
https://github.com/theforeman/foreman-documentation/issues/3335